src/components/Editor/SearchBar.js

```diff
@@ -624,7 +624,7 @@ class SearchBar extends Component {
         {
           className: classnames(className, {
             active: !symbolSearchOn && modifiers && modifiers.get(modVal),
-            disabled: symbolSearchOn
+            hide: symbolSearchOn
```

Comment:
this is a valid approach, but we often just choose not to render a component too :)

i.e.

```js
if (! symbolSearchOn) {
  return null;
}
```